buku.py

```diff
@@ -1492,6 +1492,31 @@ def get_page_title(resp):
         return htmlparser.parsed_title
 
 
+def create_poolmanager(url):
+    proxy = None
+    if url.startswith('http://') and (not os.environ.get('http_proxy') is None or
+                                      not os.environ.get('HTTP_PROXY') is None):
+        proxy = os.environ.get('http_proxy')
+        if proxy is None:
+            proxy = os.environ.get('HTTP_PROXY')
+    else if url.startswith('https://') and (not os.environ.get('https_proxy') is None or
+                                            not os.environ.get('HTTPS_PROXY') is None or
+                                            not os.environ.get('http_proxy') is None or
+                                            not os.environ.get('HTTP_PROXY') is None):
+        proxy = os.environ.get('https_proxy')
+        if proxy is None:
+            proxy = os.environ.get('HTTPS_PROXY')
+        if proxy is None:
+            os.environ.get('http_proxy')
+        if proxy is None:
+            os.environ.get('HTTP_PROXY')
+    if not proxy is None:
+        manager = urllib3.ProxyManager(proxy)
```

Comment:
Thank you!

And thanks for bringing it up. In one of my other utilities we added support for https_proxy, then another user came up with authentication support request. For `Buku` I'ld like to have a clean one-time solution which we wouldn't need to touch frequently.